docs/content/concepts/dagit/dagit.mdx

```diff
@@ -25,45 +25,31 @@ The Job page offers tools to explore a job definition and launch runs for that j
 
 The Overview tab shows the graph of ops that make up a job. Selecting an op reveals detailed metadata about it in the pane to the right.
 
-<Image
+<img
 alt="Dagit Pipeline Definition"
 src="/images/concepts/dagit/pipeline-definition.png"
-width={3808}
-height={2414}
 />
 
 ### Op Definition
 
-On the Definition tab, clicking an op, you can find detailed information about the op definition.
+On the Overview tab, clicking an op, you can find detailed information about the op definition.
 
-<Image
+<img
 alt="Dagit Solid Definition"
 src="/images/concepts/dagit/solid-definition.png"
-width={3808}
-height={2414}
 />
```

Comment:
this was intentional bc [Next Image](https://nextjs.org/docs/api-reference/next/image) has some optimization benefits and having width and height can avoid content shifting when loading images.

btw, in case you're not aware: if you write `![alt](img link)` in mdx the `make snapshot` cmd would autogen the width and height.

Reply:
> btw, in case you're not aware: if you write ![alt](img link) in mdx the make snapshot cmd would autogen the width and height.

I might not be entirely understanding how this works, but does that if I generate a new png for "/images/concepts/dagit/pipeline-definition.png", I would need to go into the docs and replace
```
<Image
alt="Dagit Pipeline Definition"
src="/images/concepts/dagit/pipeline-definition.png"
width={3808}
height={2414}
/>
```

With 
```
![alt](/images/concepts/dagit/pipeline-definition.png)
```
?

I think this makes the process of updating screenshots significantly more painful. Is there anything we can do about that?

Reply:
you're right that's a friction in the `make snapshot` script.

[1] we could do something in the mdx parser side [here](https://github.com/dagster-io/dagster/blob/56ff12be028931ce89a422ea4b388dc0c366469f/docs/next/util/imageTransformer.ts#L47) to make it easier to update the existing images while keeping the <Image /> tag.